app/src/main/java/ar/com/wolox/android/example/ui/home/news/individualNew/IndividualNewAdapterAPI.kt

```diff
@@ -0,0 +1,25 @@
+package ar.com.wolox.android.example.ui.home.news.individualNew
+
+import ar.com.wolox.android.example.network.News
+import ar.com.wolox.android.example.network.NewsServices
+import ar.com.wolox.android.example.utils.networkCallback
+import ar.com.wolox.wolmo.networking.retrofit.RetrofitServices
+import javax.inject.Inject
+
+class IndividualNewAdapterAPI @Inject constructor(private val retrofitServices: RetrofitServices) {
+
+    fun onRefreshIndividualNew(id: String, onSuccessRefresh: (News) -> Unit, onEmptyNew: () -> Unit, onFailureRefresh: () -> Unit) {
```

Comment:
It's not a good name for this. The adapter only shows messages to connect to the API, like `getNews(id)`.